prepare.sh

```diff
@@ -1,7 +1,64 @@
-#!/bin/sh
+#!/bin/bash
 
 set -x
 
 yum -y install git podman
 
-git clone --single-branch --branch=master https://github.com/ceph/ceph-csi /opt/build
+ARGUMENT_LIST=(
```

Comment:
Argument handling should happen before other "real" actions. Even if you pretty much always need to install git and podman it should happen after your parse the arguments.